components/about-page-preview.js

```diff
@@ -3,36 +3,56 @@ import { PortableText } from '@portabletext/react';
 import OverNav from './over-nav';
 import { usePreview } from '../lib/sanity.preview';
 import { aboutPagePTComponents } from '../lib/portable-text/pt-components';
+import AboutPageDropdown from './about-page-dropdown';
+const margins = 'mx-8 md:mx-10 lg:ml-10 lg:mr-0 xl:ml-20 xl:mr-0 2xl:max-w-7xl 2xl:mx-auto';
```

Comment:
super small thing but maybe a newline after the imports :)

Reply:
maybe we can also make this into global-margin type thing for previews?

Reply:
The design actually goes against the global margins. The over nav component goes to the very edge of the page for LG and XL screen sizes.

Reply:
maybe we create an alternate global margin then?